headers/fmi3FunctionTypes.h

```diff
@@ -407,13 +471,13 @@ typedef fmi3Status fmi3GetStatusTYPE       (fmi3Component c,
                                             const fmi3StatusKind s,
                                             fmi3Status* value);
 
-typedef fmi3Status fmi3GetRealStatusTYPE   (fmi3Component c,
+typedef fmi3Status fmi3GetDoubleStatusTYPE (fmi3Component c,
```

Comment:
Should this not be fmi3GetFloat64StatusTYPE? (We'll remove it later anyways if issue #484 goes the way we plan, but still).
```suggestion
typedef fmi3Status fmi3GetFloat64StatusTYPE (fmi3Component c,
```

Reply:
@pmai, should we address this here? The functions will be superseded anyway (as decided today in the Steering Committee meeting).

Reply:
I'd do so for consistency's sake, makes the other change orthogonal...